packages/orchestration/src/util/translation.ts

```diff
@@ -0,0 +1,29 @@
+import type { SAPDocumentTranslation } from '../client/api/schema/index.js';
+
+/**
+ * Convenience function to build a document translation configuration for orchestration service.
+ * @param config - Config for SAP Document Translation service.
+ * The target language is mandatory, while source language will be auto-detected if not provided.
+ * See https://help.sap.com/docs/translation-hub/sap-translation-hub/supported-languages-6854bbb1bd824ffebc3a097a7c0fd45d?state=DRAFT for list of supported languages.
+ * @returns A `SAPDocumentTranslation` configuration object.
+ * @example "buildTranslationConfig({source_language: 'de-DE', target_language: 'en-US'})"
+ */
+export function buildTranslationConfig(
+  config?: SAPDocumentTranslation['config']
```

Comment:
[pp] Can we still create a wrapper type for this? Two reasons, first we should use camel case, second if orchestration put something new in config, it won't immediately affect us when doing an orchestration spec update. This still provides us room to do adjustment.